src/newViews/AnalysisModule/index.tsx

```diff
@@ -98,8 +102,8 @@ interface AuthoringOrganizations {
 /*
 const dummyAuthoringOrganizations: AuthoringOrganizations[] = [
     { count: 10, organizationTypeId: 1, organizationTypeTitle: 'NGOs' },
-    { count: 6, organizationTypeId: 1, organizationTypeTitle: 'Government' },
-    { count: 16, organizationTypeId: 1, organizationTypeTitle: 'UN Agency' },
+    { count: 10, organizationTypeId: 1, organizationTypeTitle: 'Government' },
+    { count: 10, organizationTypeId: 1, organizationTypeTitle: 'UN Agency' },
 ];
  */
```

Comment:
any reason we are keeping this

Reply:
To test when we don't have data.
I'll remove this.